docs/api-reference/data-management-api.md

```diff
@@ -46,6 +47,22 @@ When you use these APIs concurrently with an indexing task or a kill task, the b
 Druid terminates some segments and marks others as used.
 Furthermore, it is possible that all segments could be unused, yet an indexing task might still be able to read data from these segments and complete successfully.
 
+### New Overlord APIs
+
+All of the following APIs, except [Segment deletion](#segment-deletion) have been moved from the Coordinator to the Overlord as it is the service responsible for performing actions on segment metadata on behalf of indexing tasks.
+This makes the Overlord the single source of truth for segment metadata, thus ensuring a consistent view across the Druid cluster and allowing the Overlord to cache metadata to improve performance.
+The corresponding Coordinator APIs are now deprecated and will be removed in a future release. For the time being, they simply redirect the request to the Overlord.
+The same request payloads may be used for both versions of any API and the response payload has changed only in a couple of cases.
+
+|API|Method|Deprecated Coordinator path|New Overlord path|Change in response payload|
+|---|------|---------------------------|-----------------|--------------------------|
+|[Mark a single segment as used](#mark-a-single-segment-as-used)|`POST`|`/druid/coordinator/v1/datasources/segments/{segmentId}`|`/druid/coordinator/v1/datasources/segments/{segmentId}`|Yes (old: `{"segmentStateChanged": true}`, new: `{"numChangedSegments": 1}`)|
```

Comment:
1. I believe the new overlord path should be `indexer` (instead of `coordinator`)
2. Missing `{datasource}` path parameter in the paths


```suggestion
|[Mark a single segment as used](#mark-a-single-segment-as-used)|`POST`|`/druid/coordinator/v1/datasources/{datasource}/segments/{segmentId}`|`/druid/indexer/v1/datasources/{datasource}/segments/{segmentId}`|Yes (old: `{"segmentStateChanged": true}`, new: `{"numChangedSegments": 1}`)|
```

Reply:
Ah, copy paste! Fixed now.